src/app/public/modules/numeric/numeric.service.spec.ts

```diff
@@ -134,7 +134,7 @@ describe('Numeric service', () => {
     expect(skyNumeric.formatNumber(value, options)).toBe('1.5K');
   });
 
-  it('formats 1000 as US dollar with 0 digits as $1K', () => {
+  xit('formats 1000 as US dollar with 0 digits as $1K', () => {
```

Comment:
I'm not a fan of this because we'd potentially have logic that is not tested on _any_ browser. I'd much rather reduce the number of browsers we run unit tests against, instead of not testing some aspects of the logic.

Reply:
The issue could be simply: "Add support for Microsoft Edge browser", with a short description for why we don't run unit tests against Edge. (Which would automatically be fixed once Builder supports greater versions of Angular.)